index.bs

```diff
@@ -736,6 +736,191 @@ Depending on the underlying platform, the user agent <span class=allow-2119>may<
 
 For a history and rationale of this design, please see the <a href="https://github.com/webmachinelearning/webnn/blob/master/device-selection-explainer.md">device selection explainer</a>.
 
+## Operators ## {#programming-model-operators}
+
+*This section is non-normative.*
+
+The WebNN API defines a set of operators required by well-known CNN and RNN, transformer and generative models that address key [[#usecases-application]]. The details of each operator are defined in the normative sections of this specification, in alphabetical order by the operator name. These operators are grouped into categories based on their functionality in the following non-normative table to give a functional overview of the API surface.
+
+Note: Some operators belong to multiple categories. For example, {{MLGraphBuilder/clamp()}} is both a math function and also used as an activation.
+
+<table id='operators-table' class='data'>
+  <caption>Operators by category</caption>
+  <thead>
+    <tr>
+      <th>Category</th>
+      <th>Operators​</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Tensor creation</td>
+      <td>
+      {{MLGraphBuilder/input()}},
+      {{MLGraphBuilder/constant(tensor)|constant()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Tensor rearrangement</td>
+      <td>
+      {{MLGraphBuilder/concat()}},
+      {{MLGraphBuilder/expand()}},
+      {{MLGraphBuilder/gather()}},
+      {{MLGraphBuilder/gatherElements()}},
+      {{MLGraphBuilder/scatterElements()}},
+      {{MLGraphBuilder/gatherND()}},
+      {{MLGraphBuilder/scatterND()}},
```

Comment:
Both CoreML and TOSA have a dedicated "gather/scatter" category:
https://apple.github.io/coremltools/source/coremltools.converters.mil.mil.ops.defs.html#module-coremltools.converters.mil.mil.ops.defs.iOS15.scatter_gather
https://www.mlplatform.org/tosa/tosa_spec.html#_scattergather_operators